### PR TITLE
Documentation: Example config amendment

### PR DESCRIPTION
**Pull request by vmatekole**
Minor amendment for etcd tls-setup 'hack' documentation. This may save the unsuspecting a few hrs or more ;)

### CODE REVIEW

**Comment by gyuho** [comment]
LGTM. Thanks.

File: hack/tls-setup/README.md [hunk 1]
```diff
@@ -5,5 +5,28 @@ Defaults generate an ECDSA-384 root and leaf certificates for `localhost`. etcd
 **Instructions**
 
 1. Install git, go, and make
-2. Run `make` to generate the certs
-3. Run `goreman start`
+2. Amend https://github.com/coreos/etcd/blob/master/hack/tls-setup/config/req-csr.json with IP's for each node in your cluster:
```
Comment (xiang90) [inline hack/tls-setup/README.md]: let's mention that IPs (3.8.121.201, 46.4.19.20) are the fake IPs that we add into the json.

File: hack/tls-setup/README.md [hunk 2]
```diff
@@ -5,5 +5,28 @@ Defaults generate an ECDSA-384 root and leaf certificates for `localhost`. etcd
...
+	```json
```
Comment (xiang90) [inline hack/tls-setup/README.md]: remove the indentation here for the json code block.

**Comment by xiang90** [comment]
Thanks a lot. LGTM after adding the comments on IPs.

**Comment by xiang90** [comment]
@vmatekole looks great. Can you squash two of the commits into one? So that we can get it merged?

```
git reset --soft HEAD~1
git commit --amend
```

**Comment by xiang90** [comment]
Thanks a lot!

**Pull request merged**